search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java

```diff
@@ -106,12 +104,21 @@ public static TicketParameterValidator builder() {
      * @return TicketQuery (builder pattern)
      */
     public TicketQuery applyContextAndAuthorize(RequestInfo requestInfo) throws UnauthorizedException {
+        if (Objects.isNull(requestInfo.getUserName())) {
+            throw new UnauthorizedException();
+        }
+
         var organization = requestInfo.getTopLevelOrgCristinId()
             .orElse(requestInfo.getPersonAffiliation());
+        var adminAccess = getAccessRights(requestInfo);
```

Comment:
Is admin the right word? I would think admin inherently has all ticket access rights.

```suggestion
        var ticketAccessRights = getAccessRights(requestInfo);
```

Reply:
That would be a natural thing to think, I agree, but that is wrong, Admin doesn't give you anything...